src/common/constants.hockey.ts

```diff
@@ -268,6 +268,7 @@ const AWARD_NAMES = {
 	goy: "Goalie of the Year",
 	finalsMvp: "Finals MVP",
 	allLeague: "All-League",
+	allDefensive: "All-Defense",
```

Comment:
I was thinking to not have an All-Defensive team, because of how some positions are already more offensive/defensive oriented, so the All-League team will cover it. Seems the NHL doesn't have a dedicated All-Defensive team either.